source/renderer/app/stores/BlockConsolidationStore.js

```diff
@@ -0,0 +1,141 @@
+// @flow
+import { observable, action, runInAction } from 'mobx';
+
+import Store from './lib/Store';
+import Request from './lib/LocalizedRequest';
+import {
+  BLOCK_CONSOLIDATION_IPC_REQUEST_INTERVAL,
+  BLOCK_CONSOLIDATION_API_REQUEST_INTERVAL,
+} from '../config/timingConfig';
+import { Logger } from '../utils/logging';
+import { getNumberOfEpochsConsolidatedChannel } from '../ipc/getNumberOfEpochsConsolidatedChannel';
+import type { GetConsolidatedEpochsCountMainResponse } from '../../../common/ipc/api';
+import type {
+  GetNodeSettingsResponse,
+  GetCurrentEpochFallbackResponse,
+} from '../api/nodes/types';
+
+export default class BlockConsolidationStore extends Store {
+  // Initialize store observables
+  @observable epochsConsolidated: number = 0; // Got from the IPC channel
```

Comment:
@daniloprates please change comment for this and the following line - `Got` ->  `Received` 🙏